tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -862,3 +820,59 @@ def brick_status_alert(hostname):
     finally:
         if isinstance(lock, etcd.lock.Lock) and lock.is_acquired:
             lock.release()
+
+
+def update_cluster_alert_count():
+    cluster_alert_count = 0
+    severity = ["WARNING", "CRITICAL"]
+    try:
+        alert_counts = find_volume_id()
+        alerts_arr = NS.tendrl.objects.ClusterAlert(
```

Comment:
Is it possible to load all the cluster alerts using tags only? Don't you need integration_id here? Does it default to NS.tendrl_context.integration_id in object constructor?

Reply:
@shtripat for the cluster_alert object is a subclass of alert object, so for an alert object integration_id is not a mandatory field. So for cluster alerts, integration_id is present in tags only. While forming patch cluster alert object will take integration_id from tags only. That is why i am passing only tags with integration_id.

Reply:
https://github.com/Tendrl/commons/blob/master/tendrl/commons/objects/cluster_alert/__init__.py

Reply:
ok